src/components/learner-credit-management/data/utils.js

```diff
@@ -577,20 +585,27 @@ export const hasCourseStarted = (start) => dayjs(start).isBefore(dayjs());
 export const getAssignableCourseRuns = ({ courseRuns, subsidyExpirationDatetime, isLateRedemptionAllowed }) => {
   const clonedCourseRuns = courseRuns.map(courseRun => ({
     ...courseRun,
-    enrollBy: dayjs.unix(courseRun.enrollBy).toISOString(),
+    enrollBy: courseRun.hasEnrollBy ? dayjs.unix(courseRun.enrollBy).toISOString() : null,
     upgradeDeadline: dayjs.unix(courseRun.upgradeDeadline).toISOString(),
   }));
-  const assignableCourseRunsFilter = ({ enrollBy, isActive }) => {
-    const enrollByDateThreshold = dayjs(enrollBy).isBefore(
-      Math.max(
-        dayjs(subsidyExpirationDatetime).subtract(MAX_ALLOWABLE_REFUND_THRESHOLD_DAYS, 'days').toDate(),
-        dayjs().add(DAYS_UNTIL_ASSIGNMENT_ALLOCATION_EXPIRATION, 'days').toDate(),
-      ),
-    );
-    if (isLateRedemptionAllowed) {
-      return enrollByDateThreshold;
+  const assignableCourseRunsFilter = ({
+    enrollBy, isActive, hasEnrollBy = false, isLateEnrollmentEligible = false,
+  }) => {
+    let isEligibleForEnrollment = true;
+    if (hasEnrollBy) {
+      isEligibleForEnrollment = dayjs(enrollBy).isBefore(
+        Math.min(
+          dayjs(subsidyExpirationDatetime).subtract(MAX_ALLOWABLE_REFUND_THRESHOLD_DAYS, 'days').toDate(),
+          dayjs().add(DAYS_UNTIL_ASSIGNMENT_ALLOCATION_EXPIRATION, 'days').toDate(),
+        ),
+      );
+    }
+    if (hasCourseStarted(enrollBy) && isLateRedemptionAllowed) {
```

Comment:
nit: Should we be using a function `hasCourseStarted` with an `enrollBy` date? Perhaps this util function should be renamed to be more generic, so it's not tied explicitly to "start" dates based on its name.